conda/environments/builddocs.yml

```diff
@@ -16,5 +16,15 @@ dependencies:
 - pip
 - pip:
   - --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
+  # NOTE: this 'pip install' working correctly depends on two environment variables
+  #       used by rapids-build-backend, set in the readthedocs UI
+  #
+  #       # do not try to run 'nvcc --version' to determine CUDA version
+  #       RAPIDS_DISABLE_CUDA=true
+  #
+  #       # install CUDA-12-specific dependencies
+  #       # (e.g. libucx-cu12)
+  #       RAPIDS_MATRIX_ENTRY="cuda=12.2"
+  #
```

Comment:
Thanks to @pentschev for giving me access to set environment variables in the readthedocs console for this project 😁